tools/docker/tests/test_flows.sh

```diff
@@ -0,0 +1,106 @@
+#find prplmesh folder
+scriptdir="$(cd "${0%/*}"; pwd)"
+installdir="${scriptdir%/*/*/*/*}/build/install"
+
+dbg() {
+    [ "$VERBOSE" = "true" ] && echo "$@"
+}
+
+status() {
+	printf '\033[1;35m'"$@\n"'\033[0m'
+}
+
+err() {
+    printf '\033[1;31m'"$@\n"'\033[0m'
+}
+
+success() {
+    printf '\033[1;32m'"$@\n"'\033[0m'
+}
+
+run() {
+    dbg "$*"
+    "$@" || exit $?
+}
+
+report() {
+    msg="$1"; shift
+    if "$@"; then
+        success "OK $msg"
+    else
+        err "FAIL $msg"
+        error=1
+    fi
+}
+
+start_gw_repeater() {
+    dbg "delete running containers before starting test"
+    docker rm -f $(docker ps -a -q) >/dev/null 2>&1
+    dbg "start gw and repeater"
+    ${scriptdir}/test_gw_repeater.sh >/dev/null 2>&1
+}
+
+send_topology_query() {
+    dbg "Sending toplogy query to al_mac $1"
+    docker exec -it gateway ${installdir}/bin/beerocks_cli -c "bml_trigger_topology_discovery $1" >/dev/null 2>&1
+}
+
+check_log_for_query() {
+    dbg "Confirming topology query was received"
+    docker exec -it repeater cat tmp/$USER/beerocks/logs/beerocks_agent.log | grep -i -q "Topology Query"
```

Comment:
No need for cat, just grep directly.

Reply:
```suggestion
    docker exec -it repeater grep -i -q "Topology Query" tmp/$USER/beerocks/logs/beerocks_agent.log
```

Reply:
Note - do not commit the suggestion as github offers - it adds a separate commit without DCO, so just overhead. Instead, change it in the file and add a fixup commit according to the CONTRIBUTING.md file.

Reply:
Actually, when you press "Commit suggestion" you can give the commit title, and there you can set the fixup message, i.e. "fixup! docker: add initial flow tests script".

And of course, you can anyway still move the fixup commit to the right place manually while doing git rebase -i.